composer.json

```diff
@@ -58,7 +59,7 @@
         "symfony/yaml": "^5.4 || ^6.0"
     },
     "suggest": {
-        "doctrine/annotations": "If your project use doctrine-bundle:>=2.8, this package is required to use annotations or attributes",
+        "doctrine/annotations": "If you use doctrine/doctrine-bundle >2.7, this package is required to use annotations",
```

Comment:
Incorrect wording here - the package is required to use annotations _and_ attributes - not just annotations.

Reply:
the purpose of this PR is indeed allowing the use of attributes without relying on doctrine annotations